compose/cli/main.py

```diff
@@ -273,17 +301,13 @@ def ps(self, project, options):
                 'State',
                 'Ports',
             ]
-            rows = []
-            for container in containers:
-                command = container.human_readable_command
-                if len(command) > 30:
-                    command = '%s ...' % command[:26]
-                rows.append([
-                    container.name,
-                    command,
-                    container.human_readable_state,
-                    container.human_readable_ports,
-                ])
+            rows = table(containers)
+            if orphan:
+                headers.append('Misc')
+                rows += table(orphan_containers, misc='orphan')
```

Comment:
I wonder if we need this. Thinking about `docker ps -a` it doesn't modify the columns, it just adds more rows.

I think we include this detail in the "State" column instead. Maybe as "Up (orphan)".